[skip ci] Update README.md, Link added to vLLM repository

bbeggsTT opened the pull request:
### Ticket
https://github.com/tenstorrent/vllm/issues/87

### Problem description
The vLLM and LLM sections of these two repos are important to link together because they provide helpful supplemental info.

### What's changed
Link added to vLLM repository just after the LLM table.

### Checklist
- [ ] [All post commit](https://github.com/tenstorrent/tt-metal/actions/workflows/all-post-commit-workflows.yaml) CI passes
- [ ] [Blackhole Post commit](https://github.com/tenstorrent/tt-metal/actions/workflows/blackhole-post-commit.yaml) CI with demo tests passes (if applicable)
- [ ] [Model regression](https://github.com/tenstorrent/tt-metal/actions/workflows/perf-models.yaml) CI passes (if applicable)
- [ ] [Device performance regression](https://github.com/tenstorrent/tt-metal/actions/workflows/perf-device-models.yaml) CI passes (if applicable)
- [ ] (For models and ops writers) [Single-card demo tests](https://github.com/tenstorrent/tt-metal/actions/workflows/single-card-demo-tests.yaml) CI passes (if applicable) See [recommended dev flow](https://github.com/tenstorrent/tt-metal/blob/main/models/MODEL_ADD.md#a-recommended-dev-flow-on-github-for-adding-new-models).
- [ ] New/Existing tests provide coverage for changes

bbeggsTT committed:
Update README.md, Link added to vLLM repository

Link added to vLLM repository just after the LLM table.

skhorasganiTT commented on README.md:
```diff
@@ -48,6 +48,9 @@
 | [Mistral 7B](./models/tt_transformers)                        | 32    | [n150](https://tenstorrent.com/hardware/wormhole)        |  104 | 26.4  | 23     | 844.8 | [v0.59.0-rc13](https://github.com/tenstorrent/tt-metal/tree/v0.59.0-rc13)    |  [739dcaa](https://github.com/tenstorrent/vllm/tree/739dcaa2915fa29d757c25a02c17aadce0c58055) |
 | [Mixtral 8x7B (TP=8)](./models/demos/t3000/mixtral8x7b)       | 32    | [QuietBox](https://tenstorrent.com/hardware/tt-quietbox) | 207       | 16.6  | 33              | 531.2  | [v0.58.0-rc22](https://github.com/tenstorrent/tt-metal/tree/v0.58.0-rc22) |                                                                                                   |
 
+>[!NOTE]
+>For more information regarding vLLM installation, environment creation, model versions, model weights, and more, visit the [Tenstorrent vLLM repository](https://github.com/tenstorrent/vllm/blob/dev/tt_metal/README.md).
+
```
vLLM does not have specific instructions for model versions and model weights. Also we have a list of "Notes" below, can you add this as a bullet point at the bottom of the existing notes and change it to something like "For more information regarding running models on Tenstorrent hardware with vLLM, please visit ..."

bbeggsTT committed:
Update README.md

Link to vLLM repo moved to bottom of notes section. Reference to model versions and weights removed.

bbeggsTT committed:
Merge branch 'main' into bbeggsTT-patch-7
